poetry/core/packages/directory_dependency.py

```diff
@@ -121,7 +122,8 @@ def base_pep_508_name(self) -> str:
         if self.extras:
             requirement += "[{}]".format(",".join(self.extras))
 
-        requirement += " @ {}".format(self._path.as_posix())
+        path_url = path_to_url(self._path)
```

Comment:
For consistency's sake I would say we do exactly this > https://github.com/python-poetry/poetry-core/blob/master/poetry/core/packages/file_dependency.py#L110